Java/GUI.java

```diff
@@ -130,6 +130,14 @@ public void drawMove(char symbol, int row, int col)
             g.drawOval(xCoordinate - 25, yCoordinate - 25, 50, 50);
         }
     }
+    
+    /**
+     * Apologize because we can do nothing more. 
+     */
+    public void tellTheUserWeAreSorry() 
+    {
+        JOptionPane.showMessageDialog(null, "I fucked up bad, and I am sorry.");
+    }
```

Comment:
Let the user know we had a major error in the program and don't really feel like trying to fix it.